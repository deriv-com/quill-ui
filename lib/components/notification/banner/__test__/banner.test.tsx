import { render } from "@testing-library/react";
import NotificationBanner from "..";

describe("NotificationBanner", () => {
    it("should have correct classname", () => {
        const { container } = render(<NotificationBanner className="test" />);

        expect(container).toMatchSnapshot();
    });
});
