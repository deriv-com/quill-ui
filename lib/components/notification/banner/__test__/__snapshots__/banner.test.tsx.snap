// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotificationBanner should have correct classname 1`] = `
<div>
  <div
    class="notification-banner test"
  >
    <div
      class="body"
    >
      <div
        class="icon"
      >
        (i)
      </div>
      <div
        class="content"
      >
        <p
          class="quill-typography__body-text__size--md__weight--regular__decoration--default quill-typography__color--default title"
        >
          Title
        </p>
        <p
          class="quill-typography__body-text__size--md__weight--regular__decoration--default quill-typography__color--default message"
        >
          Message
        </p>
      </div>
    </div>
    <button
      class="close-button"
    >
      x
    </button>
  </div>
</div>
`;
